scripts/dev/dockerutil.py

```diff
@@ -28,6 +28,42 @@ def push_image(tag: str) -> None:
         print("\r" + push_image_formatted(line), end="", flush=True)
 
 
+def retag_image(
+    old_repo_url: str,
+    new_repo_url: str,
+    old_tag: str,
+    new_tag: str,
+    path: str,
+    labels: Optional[dict] = None,
+    username: Optional[str] = None,
+    password: Optional[str] = None,
+    registry: Optional[str] = None,
+) -> None:
+    with open(f"{path}/Dockerfile", "w") as f:
+        f.write(f"FROM {old_repo_url}:{old_tag}")
+    client = docker.from_env()
+    if username is not None and password is not None and registry is not None:
+        client.login(username=username, password=password, registry=registry)
+
+    i, _ = client.images.build(path=f"{path}", labels=labels, tag=new_tag)
```

Comment:
[nit] we should give this a proper name like `image`
[q] what is the second value we're ignoring?

Reply:
From the docs: ` The second item is a generator of the build logs as JSON-decoded objects.` Do we want to use them to output logs or are we ok with ignoring it?

Reply:
seems okay to ignore it in this case!